fish_prompt.fish

```diff
@@ -13,7 +13,7 @@ function fish_prompt
 	__sf_util_set_default SPACEFISH_PROMPT_SUFFIXES_SHOW true
 	__sf_util_set_default SPACEFISH_PROMPT_DEFAULT_PREFIX "via "
 	__sf_util_set_default SPACEFISH_PROMPT_DEFAULT_SUFFIX " "
-	__sf_util_set_default SPACEFISH_PROMPT_ORDER time user dir host git package node ruby golang php rust haskell pyenv kubecontext exec_time line_sep battery jobs exit_code char
+	__sf_util_set_default SPACEFISH_PROMPT_ORDER time user dir host git package node docker ruby golang php rust haskell pyenv kubecontext exec_time line_sep battery jobs exit_code char
```

Comment:
This is just to make sure we have the same prompt section ordering as [spaceship does](https://github.com/denysdovhan/spaceship-prompt/blob/c5ed80a65b8693f98f6fc8f783ce228efd8eb618/spaceship.zsh#L50). 🚀 

```suggestion
	__sf_util_set_default SPACEFISH_PROMPT_ORDER time user dir host git package node ruby golang php rust haskell docker pyenv kubecontext exec_time line_sep battery jobs exit_code char
```